.github/workflows/chromatic.yml

```diff
@@ -2,7 +2,14 @@ name: Chromatic
 
 on:
   workflow_dispatch:
+  pull_request:
+    branches:
+      - main
```

Comment:
This should be `"**"` so it runs on all pull requests. The rest of it looks good.